apps/server/src/modules/management/seed-data/Accounts.ts

```diff
@@ -0,0 +1,691 @@
+		activated: true,
+	},
+	{
+		_id: '5fa31f99b229544f2c697e62',
+		username: 'fernando.admin.qa@schul-cloud.org',
+		userId: '5fa31f99b229544f2c697e5c',
+		password: '$2a$10$MocihKfsY2t2RziBvodRHux96wvdjpNxD2k1D2hh0OgsTxpC2sJ..',
+		createdAt: '2020-11-04T21:39:37.850Z',
+		updatedAt: '2020-11-04T21:54:23.661Z',
+		lasttriedFailedLogin: '1970-01-01T00:00:00Z',
+		activated: true,
+	},
+	{
+		_id: '5fa32008b229544f2c697eea',
+		username: 'daniel.teacher.qa@schul-cloud.org',
+		userId: '5fa32008b229544f2c697ee4',
+		password: '$2a$10$AF9JmqNJrtSjDjJZeFmUMucTgsIZ8fcXw3GNQvRFiEb0eFfLCMWcG',
+		createdAt: '2020-11-04T21:41:28.965Z',
+		updatedAt: '2020-11-04T21:55:03.253Z',
+		lasttriedFailedLogin: '1970-01-01T00:00:00Z',
+		activated: true,
+	},
+	{
+		_id: '5fa3206ab229544f2c697f4d',
+		username: 'dani.alvesihno.qa.s@schul-cloud.org',
+		userId: '5fa3206ab229544f2c697f47',
+		password: '$2a$10$2UP8LFYhQCBUE/CkCBqtGuEUy8UrFoU7sXeJo0DysYcdaIoOXcHx6',
+		createdAt: '2020-11-04T21:43:06.328Z',
+		updatedAt: '2020-11-04T21:54:40.463Z',
+		lasttriedFailedLogin: '1970-01-01T00:00:00Z',
+		activated: true,
+	},
+	{
+		_id: '5fa321abb229544f2c698068',
+		username: 'renato.delima.qa.s@schul-cloud.org',
+		userId: '5fa321abb229544f2c698062',
+		password: '$2a$10$owiq/gplseyDaiCQgCq1N.XOIqUFiHSy7KjfeC.lNwTyG/GzOXXZ.',
+		createdAt: '2020-11-04T21:48:27.261Z',
+		updatedAt: '2020-11-04T21:55:23.346Z',
+		lasttriedFailedLogin: '1970-01-01T00:00:00Z',
+		activated: true,
+	},
+	{
+		_id: '5fa32210b229544f2c6980db',
+		username: 'carlota.vargas.qa.s@schul-cloud.org',
+		userId: '5fa32210b229544f2c6980d5',
+		password: '$2a$10$6GoMpAjtqZ/lsJ9se57iV.yZhTRsGx.y1LVOe9ijJ565CWnhyvar.',
+		createdAt: '2020-11-04T21:50:08.665Z',
+		updatedAt: '2020-11-04T21:59:03.439Z',
+		lasttriedFailedLogin: '1970-01-01T00:00:00Z',
+		activated: true,
+	},
+	{
+		_id: '5fa322b1b229544f2c69814e',
+		username: 'julio.expert.qa.e@schul-cloud.org',
+		userId: '5fa322b1b229544f2c698148',
+		password: '$2a$10$V2p/57q4etJnft2dY4q89.xR3W2QfZnAAY4fYOZ37zYJgrHl2NIYu',
+		createdAt: '2020-11-04T21:52:49.588Z',
+		updatedAt: '2020-11-05T06:47:45.652Z',
+		lasttriedFailedLogin: '1970-01-01T00:00:00Z',
+		activated: true,
+	},
+	{
+		_id: '0000d213816abba584714c00',
+		username: 'admin.extern@schul-cloud.org',
+		userId: '0000d213816abba584710000',
+		password: '$2a$10$wMuk7hpjULOEJrTW/CKtU.lIETKa.nEs8fncqLJ74SMeX.fzJXBla',
+		createdAt: '2017-01-01T00:06:37.148Z',
+		updatedAt: '2017-01-01T00:06:37.148Z',
+	},
+	{
+		_id: '6335405a1f94d7fc9fc91861',
+		username: 'expert@schul-cloud.org',
+		userId: '63354059dc728d001f53fb6d',
+		password: '$2a$10$wMuk7hpjULOEJrTW/CKtU.lIETKa.nEs8fncqLJ74SMeX.fzJXBla',
+		createdAt: '2022-09-29T06:51:06.228Z',
+		updatedAt: '2022-09-29T06:51:06.228Z',
+
+		activated: true,
+	},
+];
+
+export const Accounts = accountPartials.map((partial) => {
+	const account = accountFactory.buildWithId(
+		{
+			activated: partial.activated,
+			userId: partial.userId,
```

Comment:
You must be sure that the userId is connected to the real existing user now and in future. Maybe UserAndAccountTestFactory can solve this for you and avoid work. But the school must be created before and passed.

Reply:
Is this not the case? You replace the school.json, account.json and user.json and must connect and execute this.

Reply:
Okay, I will rewrite this part to create school, account and user in a single go.